src/Adapter/Module/ModulePresenter.php

```diff
@@ -67,6 +68,22 @@ public function present($module)
         );
     }
 
+    /**
+     * Transform a collection of addons as a simple array of data.
+     *
+     * @param array $modules
+     * @return array
+     */
+    public function presentCollection(array $modules)
```

Comment:
it's not a collection, looking at the function name I expect ``AddonsCollection`` to be used here instead of an array: is it possible?